src/playwright/transformPlaywrightJson.ts

```diff
@@ -0,0 +1,63 @@
+import * as t from '@babel/types';
+import generate from '@babel/generator';
+import { toId } from '@storybook/csf';
+
+import { testPrefixer } from './transformPlaywright';
+
+type Story = { id: string; name: string; title: string; parameters?: Record<string, any> };
+
+const makeTest = (story: Story): t.Statement => {
+  const result: any = testPrefixer({
+    name: t.stringLiteral(story.name),
+    title: t.stringLiteral(story.title),
+    id: t.stringLiteral(story.id),
+    // FIXME
+    storyExport: t.identifier(story.id),
+    hasPlayFn: t.booleanLiteral(false),
+  });
+  const stmt = result[1] as t.ExpressionStatement;
+  return t.expressionStatement(
+    t.callExpression(t.identifier('it'), [t.stringLiteral('test'), stmt.expression])
+  );
+};
+
+const makeDescribe = (title: string, stmts: t.Statement[]) => {
+  return t.expressionStatement(
+    t.callExpression(t.identifier('describe'), [
+      t.stringLiteral(title),
+      t.arrowFunctionExpression([], t.blockStatement(stmts)),
+    ])
+  );
+};
+
+/**
+ * Generate one test file per component so that Jest can
+ * run them in parallel.
+ */
+export const transformPlaywrightJson = (src: string) => {
+  const json = JSON.parse(src);
+  if (json.v !== 3) {
+    throw new Error(`Unsupported version ${json.v}`);
+  }
+  const stories = Object.values(json.stories) as Story[];
+  const titleIdToStories = stories.reduce((acc, story) => {
+    if (!story.parameters?.docsOnly) {
```

Comment:
Is this dependant on Storybook version?

I don't have it in MealDrop and it uses 6.4.14:
https://mealdrop.vercel.app/storybook/stories.json

Running this worked well until it timed out in the docs only stories:
`TARGET_URL=https://mealdrop.vercel.app/storybook/ yarn test-storybook:playwright-json --verbose`

![image](https://user-images.githubusercontent.com/1671563/151410337-1ec5a569-d91a-4800-b39e-9c0f6b160601.png)

Reply:
Great catch. This was a v6-compatibility feature and is not included in the v7 store. I've updated the code.